llama_index/storage/docstore/types.py

```diff
@@ -58,6 +68,19 @@ def set_document_hash(self, doc_id: str, doc_hash: str) -> None:
     def get_document_hash(self, doc_id: str) -> Optional[str]:
         ...
 
+    # ==== Ref Docs =====
+    @abstractmethod
+    def get_all_ref_doc_info(self) -> Optional[Dict[str, RefDocInfo]]:
```

Comment:
curious: what do we need this interface for?

I think ideally I want to limit the amount of "fetch all" calls, since it could be expensive with a document store backed by a remote connection.

Reply:
This gives the user an easy way to see the documents that have been ingested into the docstore